workflow/apm/services.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Services
+sidebarTitle: Services
+---
+
+The Services page displays all of your APM data in a unified view. A service groups together interconnected endpoints, queries, or jobs within your application. This is useful for contextualizing broader issues within your software infrastructure. 
+
+This landing page displays common metrics like requests, latency, and errors. Use the __Resources__, __Traces__, and __Logs__ tabs to drill into your data. This page is available across other APM and log pages to easily understand how data is related to other aspects of your application. Clicking on any of the charts or tables within this page will navigate you to a deeper view of that selected service, including resources, traces, and metrics.
+
+# Resources
+
+Resources are all of the endpoints being requested for a given service, such as CPU, memory, and network bandwidth. 
+
+The Resources Table displays all of your APM services in a unified table format. Thisd table is useful for drilling down into the endpoints you are investigating to find the root cause of issues related to the services you are investigating.
```

Comment:
This section is a little confusing because of the use of the word "investigating" twice, and also that you state it displays all of your APM Services which it does not. This page is drilled into a single service at a time, so the resources section is only for the specific service drilled into. Let's discuss in our 1-1 tomorrow.